osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -195,15 +196,32 @@ def execute(extraction_order_id, execution_configuration):
     extraction_order.state = models.ExtractionOrderState.WAITING
     extraction_order.save()
 
+    result_directory_name = str(extraction_order.id) + '_' + NAME
+    result_directory_path = os.path.join(settings.RESULT_MEDIA_ROOT, result_directory_name)
+
     with tempfile.TemporaryDirectory() as tmp_dir:
         original_cwd = os.getcwd()
 
         try:
-            shutil.copyfile(
-                os.path.join(os.path.dirname(__file__), 'blackbox', 'docker-compose-conversion-blackbox.yml'),
-                os.path.join(tmp_dir, 'docker-compose.yml')
+            compose_file_path = os.path.join(
+                os.path.dirname(__file__), 'blackbox', 'docker-compose-conversion-blackbox.yml'
             )
+            with open(compose_file_path, "r") as compose_file:
+                compose_template = compose_file.read()
+
             os.chdir(tmp_dir)
+
+            # create a directory for result files of the order -> prevent mess with files of different orders
+            # file collection for appending OutputFiles to ExtractionOrder may be interrupted
+            # '-> files of an order will be collected by an other order
```

Comment:
This comment doesn't make clear what _will_ happen (with the current, fixed state of the code) and what _would_ happen if it'd been done wrong (as before.)